src/test/java/org/folio/marccat/resources/SearchTest.java

```diff
@@ -113,7 +113,7 @@ public void searchAuth_return200Status() {
     String url = getURI("/marccat/searchAuth");
     Map<String, String> headers = addDefaultHeaders(url, StorageTestSuite.TENANT_ID);
 
-    given().param("q", "NA giannini").param("lang", "eng").headers(headers).when().get(url).then().statusCode(200);
+    given().param("q", "NA IbaiScanbit").param("lang", "eng").headers(headers).when().get(url).then().statusCode(200);
```

Comment:
Format this code

Reply:
It will change in other PR with all the formatted code